src/ZEO/TransactionBuffer.py

```diff
@@ -23,7 +23,8 @@
 
 import tempfile
 
-from ZEO._compat import Pickler
+from zodbpickle.pickle import Pickler
+
 from ZEO._compat import Unpickler
```

Comment:
We could use the standard `Unpickler` here.

Reply:
This is because `noload` is not used here, right ?

Reply:
The only difference between `zodbpickle.pickle.Unpickler` and `_compat.Unpickler` is the possibility to override `find_global`. This is used in `asyncio.marshal` but not in `TransactionBuffer`.

Reply:
ah I see, I thought by "standard `Unpickler`" you meant `pickle.Unpickler`, but you meant `zodbpickle.pickle.Unpickler`.

Reply:
Why haven't we updated `zodbpickle.Unpickler` to include the `find_global` support, rather than subclassing it here?